### DISCUSSION TITLE
readme file is outdated.

### DISCUSSION THREAD

**Discussion by Frans06** [sergiodxa/remix-i18next#198] [Q&A] [answered]
I tried to follow the example but it seems to be outdated. The import lines from 'remix-i18next/client' and  'remix-i18next/server' don't work with the remix version 2.9.2.  for the 6.1.0 version. I think the newly exported package is missing an index.js export file. But also I'm quite sure I just couldn't figure out how to import the package modules.

**Comment by sergiodxa** [comment, marked as answer]
Those imports are correct, here's an example app https://github.com/sergiodxa/remix-vite-i18next.

Here's `remix-i18next/client` https://github.com/sergiodxa/remix-vite-i18next/blob/0a9e99a200418f1c99dd9102d51928dda5114776/app/entry.client.tsx#L8.

Here's `remix-i18next/server` https://github.com/sergiodxa/remix-vite-i18next/blob/0a9e99a200418f1c99dd9102d51928dda5114776/app/modules/i18n.server.ts#L2

Here's `remix-i18next/react` https://github.com/sergiodxa/remix-vite-i18next/blob/0a9e99a200418f1c99dd9102d51928dda5114776/app/root.tsx#L12

Double check your configuration against that app, one common issue is the `tsconfig.json` file.
